### PR TITLE
Add `.contact.urn_display` to runs endpoint

### PR DESCRIPTION
**Pull request by rowanseymour**
Will add this in other places but this PR is complicated enough for now

```json
{
  "uuid": "9a00cb2f-b63b-4796-bc17-e1939fd29303",
  "name": "Frank",
  "urn": "twitterid:2352463463#franky",
  "urn_display": "franky"
}
```

if org is anon...

```json
{
  "uuid": "9a00cb2f-b63b-4796-bc17-e1939fd29303",
  "name": "Frank",
  "urn": "twitterid:********",
  "urn_display": "0000023525",
}
```

if contact doesn't have a URN...

```json
{
  "uuid": "9a00cb2f-b63b-4796-bc17-e1939fd29303",
  "name": "Frank",
  "urn": null,
  "urn_display": null,
}
```

but I have a sense that @ericnewcomer is going to want the anon id even when there's no URN so starting to question stuffing it in there

### CODE REVIEW

File: temba/api/v2/views.py [hunk 1]
```diff
@@ -1428,7 +1428,6 @@ def filter_queryset(self, queryset):
 
         # use prefetch rather than select_related for foreign keys to avoid joins
         queryset = queryset.prefetch_related(
-            Prefetch("org"),
```
Comment (rowanseymour) [inline temba/api/v2/views.py:1431]: `Contact.bulk_urn_cache_initialize` now sets request org on all contacts and urns

File: temba/contacts/tests.py [hunk 2]
```diff
@@ -2117,7 +2117,7 @@ def omnibox_request(query, version="1"):
         # error is swallowed and we show no results
         self.assertEqual([], omnibox_request("search=-123`213"))
 
-        with self.assertNumQueries(16):
+        with self.assertNumQueries(15):
```
Comment (rowanseymour) [inline temba/contacts/tests.py:2120]: because we're not prefetching org but using the request org

File: temba/tests/base.py [hunk 3]
```diff
@@ -766,6 +766,8 @@ def setUp(self):
             role.group  # noqa
             role.permissions  # noqa
 
+        self.maxDiff = None
```
Comment (rowanseymour) [inline temba/tests/base.py:769]: unrelated but am always adding this to failing tests and not sure why we don't just always have it set

**Comment by rowanseymour** [comment]
Updated... 

```json
{
  "uuid": "9a00cb2f-b63b-4796-bc17-e1939fd29303",
  "name": "Frank",
  "urn": "twitterid:2352463463#franky",
  "urn_display": "franky"
}
```

if org is anon...

```json
{
  "uuid": "9a00cb2f-b63b-4796-bc17-e1939fd29303",
  "name": "Frank",
  "urn": "twitterid:********",
  "urn_display": "********"
  "anon_display": "0000023525",
}
```

if contact doesn't have a URN...

```json
{
  "uuid": "9a00cb2f-b63b-4796-bc17-e1939fd29303",
  "name": "Frank",
  "urn": null,
  "urn_display": null,
}
```

**Review by norkans7 — APPROVED** [review]

**Pull request merged**